styles/rules.css

```diff
@@ -0,0 +1,51 @@
+body {
+  height: 100vh;
+}
+
+.header {
+  background: var(--primary-color);
+}
+
+.rules-container {
+  margin:2rem auto;
+  height: 60vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.rules {
+  border: 2px solid var(--primary-color);
+  width: 40vw;
+  padding: 1rem;
+}
+
+@media(max-width:800px){
+    .rules{
+        width: 80vw;
+    }
+}
+
+.list-rules {
+  margin: 10px 0;
+  font-size: 1.2rem;
+}
+
```

Comment:
Try not to write media queries in between your css file. Keep it either at the top or the bottom.

Reply:
done